src/evidently/types.ts

```diff
@@ -0,0 +1,54 @@
+export type EvaluationCallback = (err?: Error, res?: EvaluationResult) => void;
+
+export type ContextType = { [k: string]: any };
+export type PartialEvidentlyConfig = {
+    project: string;
+    endpoint?: string;
+};
+export type EvidentlyConfig = {
+    project?: string;
+    endpoint: string;
+    endpointUrl: URL;
+};
+export type InitializeFeaturesRequest = {
+    features: string[];
+    entityId?: string;
+    context?: ContextType;
+};
+type VariationValue = {
+    boolValue?: boolean;
+    doubleValue?: number;
+    longValue?: number;
+    stringValue?: string;
+};
+export type EvaluationResult = {
```

Comment:
Based on other implementations, the result type should include `EvaluateFeature` in it: `EvaluateFeatureResult`, `EvaluateFeatureOutput`, etc